File: Tests.FSharp/LiteralTests.fs
namespace Tests.FSharp

open System
open System.Text.Patterns
open Microsoft.VisualStudio.TestTools.UnitTesting

[<TestClass>]
type LiteralTests() =
    [<TestMethod>]
    member this.Constructor() =
        let defaultComparison = p"Hello"
        ()

File: Patterns.Parser.Tests/LiteralTests.cs
using Microsoft.VisualStudio.TestTools.UnitTesting;
using Stringier.Patterns;
using Stringier.Patterns.Parser;

namespace Patterns.Parser.Tests {
	[TestClass]
	public class LiteralTests {
		[TestMethod]
		public void Consume() {
			Source source = new Source("\"hello\"");
			Assert.AreEqual("hello", Literal.Consume(ref source).ToString());
		}
	}
}
